docs/modules/general/main.mdx

```diff
@@ -155,10 +158,34 @@ Additionally, a variant can also contain constants, which allows you to define t
 
 | Attribute | Description | Value |
 |---|---|---|
-| `variant` | The name of the variant to target. | <span className="badge badge--primary">Variant ID</span> |
+| `variant` | The map variant to target.<br />*Multiple variants can be targeted as long as it is separated with a comma (`,`).* | <span className="badge badge--primary">Variant ID</span> |
 | `has-variant` | Target all maps with a specified variant.<br />**Note:** This can be useful in a server's global XML file. | <span className="badge badge--primary">Variant ID</span> |
-
-**Note:** Multiple variants can be targeted as long as it is separated with a comma (`,`).
+| `min-server-version` | The minimum server version this variant can run on.<br />**Note:** PGM will attempt to detect the map version from its `level.dat` file. |
+| `max-server-version` | The maximum server version this variant can run on. |
```

Comment:
referencing variants in `if` min-server-version makes little sense
also the note here makes no sense

Reply:
> referencing variants in `if` min-server-version makes little sense

These also show up in Map attributes table as well. Are there no map variants that would be using a different version than the original map?

Reply:
The thing is simply that `if` is deciding if this block of xml applies or not based on the version. Variant is irrelevant
```xml
<if min-server-version="1.9">
  <item>shield</item>
</if>
```
this is purely a version dependant line of xml, that has NO relation to variants whatsoever. The map can play in 1.8 and shields won't be referenced, or if it plays in 1.9+ then shields are mentioned (imagine this being an itemremove or whatever)